megalista_dataflow/setup.py

```diff
@@ -20,10 +20,16 @@
     author='Google',
     author_email='megalista-admin@google.com',
     url='https://github.com/google/megalista/',
-    install_requires=['google-ads==17.0.0', 'google-api-python-client==2.65.0',
-                      'google-cloud-bigquery==2.34.3','aiohttp==3.6.2',
-                      'google-cloud-storage==2.2.1', 'google-cloud-firestore==2.4.0',
-                      'pyparsing==2.4.7', 'proto-plus==1.19.6', 'protobuf==3.20.0',
-                      'pandas==1.1.5', 'boto3==1.21.4'],
+    install_requires=[
+        'google-ads==17.0.0', 
+        'google-api-python-client==2.77.0',
+        'google-cloud-bigquery==2.34.3',
+        'aiohttp==3.8.3',
+        'google-cloud-storage==2.2.1',
+        'google-cloud-firestore==2.4.0',
+        'protobuf==3.20.0',
+        'pandas==1.5.3', 
+        'boto3==1.21.4'
```

Comment:
Hi Pedro, quick comment here, the boto3 version in this file does not match the version in the requirements.txt file, could you please update it?

Thank you!